<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(229, 229, 229);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Listen">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);border:1px groove gray;border-radius:4px;font: 20 14pt &quot;Times New Roman&quot;;padding:1px 4px;border-style: outset;}


                                       
QPushButton:hover{background-color:#79ff50; color: black;}
                                    
QPushButton:pressed{background-color:rgb(204, 228, 247);border-style: inset;</string>
    </property>
    <property name="text">
     <string>Monitor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serverSend">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>650</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);border:1px groove gray;border-radius:4px;font: 20 14pt &quot;Times New Roman&quot;;padding:1px 4px;border-style: outset;}


                                       
QPushButton:hover{background-color:#79ff50; color: black;}
                                    
QPushButton:pressed{background-color:rgb(204, 228, 247);border-style: inset;</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 10px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;
   </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>431</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 10px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;
   </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="port">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
 
	border:2px solid#7b7b7b;
	border-radius:8px;
	background-color:#fffef0
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>端口：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Server Send:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Server Receive:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="messageList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>431</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 10px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;
   </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>聊天记录：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>497</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
